### DISCUSSION TITLE
Can we fetch AKS metrics from Insights tab using promitor

### DISCUSSION THREAD

**Discussion by HusnainTaja** [tomkerkhove/promitor#2397] [Q&A]
Hi,

I have a kubernetes cluster on AKS.There is one insights tab from which we want to collect different types of metrics i.e cpu utilization,memory_utilization for each pod, containers metrics etc.

Can we use promitor for this ? Does promitor supports scraping of this metrics ?

![image](https://github.com/tomkerkhove/promitor/assets/49810774/6155a8e1-ed2e-481d-9bdf-20eaeedc818c)

**Comment by tomkerkhove** [comment]
If these are in Azure Monitor, then yes. But I don't know where these come from

**Comment by Charlotte-br560** [comment]
Yes, the Promitor supports scraping AKS metrics from the Insights tab. You can use Promitor to collect various metrics such as CPU utilization, memory utilization for each pod, and container metrics. It provides a convenient solution for fetching and monitoring these metrics within your Kubernetes cluster on AKS.
